spec/lib/azure_blob_store_spec.rb

```diff
@@ -149,4 +149,28 @@ def assert_path(path, expected)
       assert_path("https://hello", nil)
     end
   end
+
+  describe ".url_for" do
+    def assert_url(url, expected)
```

Comment:
This method doesn't seem to be used. Was it left over by mistake after some debugging?

Reply:
Oops, yep. Good catch!